docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3.9'
+
+services:
+
+  db:
+    image: postgres:14-alpine
+    container_name: dbCont
+    restart: always
+    # use different exposed port from "5432" to anything else because "5432" is desired for the pgAdmin
+    ports:
+      - "5555:5432"
```

Comment:
**Tips to solve the problem of connecting to dockerized postgres db :**
- Use different exposed port instead of `5432` because it is desired for the pgAdmin localserver.
-